docs/Privacy_policy.md

```diff
@@ -1,3 +1,67 @@
 # Privacy policy
 
-TODO
+This Privacy Policy describes our policies and procedures on the collection, use and disclosure of Your information when You use the Service and tells You about Your privacy rights and how the law protects You.
+We use Your Personal data to provide and improve the Service. By opting in, You agree to the collection and use of information in accordance with this Privacy Policy.
+
+# Interpretation and Definitions
+## Interpretation
+The words of which the initial letter is capitalized have meanings defined under the following conditions. The following definitions shall have the same meaning regardless of whether they appear in singular or in plural.
+
+## Definition
+For the purposes of this Privacy Policy:
+* __Organisation__ (referred to as either "We", "Us" or "Our" in this Agreement) refers to The IAI Group
+* __Bot__ refers to the reccomendation bot you interract with
+* __Service__ refers to the Bot.
+* __You__ means the individual accessing or using the Service, or the organisation, or other legal entity on behalf of which such individual is accessing or using the Service, as applicable.
+* __Personal Data__ is any information that relates to an identified or identifiable individual.
+* __Usage Data__ refers to data collected automatically, either generated by the use of the Service or from the Service infrastructure itself.
+
+# Collecting and Using Your Personal Data
+## Types of Data Collected
+### Personal Data
+While using Our Service, We may ask You to provide Us with certain personally identifiable information that can be used to contact or identify You. Personally identifiable information may include, but is not limited to:
+* PSID
+* Messages
+
+### PSID
+PSID is an ID assigned to you when you first initiate a conversation with the bot, which is permanent, and specific to the Bot's Facebook Page MovieBot. This is stored along with the conversation messages to identify you.
+
+### Messasges
+Messages are all messages exchanged with the bot, both sent by you and the bot.
+
+# Your Pesonal Data
+## Use of Your Personal Data
+The Bot will use Personal Data for the purpose of improving the reccomendations, making them more accurate.
```

Comment:
"improving the Service to make recommendations that are believed to be more relevant to You."